### PR TITLE
ISPN-5365 JmxManagementIT.testCacheStatisticsAttributes always fails

### PR DESCRIPTION
**Pull request by vblagoje**
Master only. See https://issues.jboss.org/browse/ISPN-5365 for more detals. @danberindei please review and integrate.

### CODE REVIEW

File: core/src/main/java/org/infinispan/interceptors/CacheMgmtInterceptor.java [hunk 1]
```diff
@@ -356,6 +356,18 @@ public long getTimeSinceStart() {
    }
 
    @ManagedAttribute(
+         description = "Number of seconds since cache started",
+         displayName = "Seconds since cache started",
+         units = Units.SECONDS,
+         measurementType = MeasurementType.TRENDSUP,
+         displayType = DisplayType.SUMMARY
+   )
+   public long getElapsedTime() {
```
Comment (tristantarrant) [inline core/src/main/java/org/infinispan/interceptors/CacheMgmtInterceptor.java]: Mark this as deprecated and let's remove it in 8.0

Comment (vblagoje) [inline core/src/main/java/org/infinispan/interceptors/CacheMgmtInterceptor.java]: Ok will do so

Comment (vblagoje) [inline core/src/main/java/org/infinispan/interceptors/CacheMgmtInterceptor.java]: Marked as deprecated @danberindei @tristantarrant Please verify and integrate.

**Comment by galderz** [comment]
Integrated, thanks!

**Pull request closed without merging**